src/main/java/com/EstimatedDeliveryDate/Entity/CutOffTimeList.java

```diff
@@ -0,0 +1,30 @@
+package com.EstimatedDeliveryDate.Entity;
+
+import java.io.Serializable;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBDocument;
+
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/*Document object for the CutOffTimeList*/
+@Setter
+@NoArgsConstructor
+@DynamoDBDocument
+public class CutOffTimeList implements Serializable {
```

Comment:
Similar to holiday, cutofftimelist does not make sense here.
The domain object suggests its only cutofftime.